src/parser.cpp

```diff
@@ -36,4 +37,118 @@ types::VPDMapVariant Parser::parse()
     return l_parser->parse();
 }
 
+int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
+{
+    // Update keyword's value on hardware
+    int l_bytesUpdatedOnHardware;
```

Comment:
Good practice to initialize the variable. Better initialize with -1 at the time of declaration itself.